lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -1138,14 +1139,16 @@ my %stats = (
         CALC => sub {
             my ($self, $sql) = @_;
 
-            my $data = $sql->select_list_of_lists(
-                'SELECT type.id, COUNT(rg.id) AS count
-                 FROM release_group_secondary_type type
-                 LEFT JOIN release_group_secondary_type_join type_join
```

Comment:
I can't see why this used to be a left join, can you?

Reply:
I suppose in case a secondary type is completely unused, it'll log 0 uses?  Probably only relevant on standalone/test DBs.

Reply:
I guess that might be it. I'll make it a left join again then I guess.